packages/loot-core/src/server/budget/goalsSchedule.ts

```
@@ -29,11 +29,21 @@
       extractScheduleConds(conditions);
-    const scheduleAmount =
+    let scheduleAmount =
       amountCondition.op === 'isbetween'
         ? Math.round(amountCondition.value.num1 + amountCondition.value.num2) /
           2
         : amountCondition.value;
+    // Apply adjustment percentage if specified
+    if (template[ll].adjustment) {
+      // Validate adjustment bounds (e.g., -100% to +1000%)
+      if (template[ll].adjustment <= -100 || template[ll].adjustment > 1000) {
+        errors.push(`Invalid adjustment percentage for schedule ${template[ll].name}`);
```

ESLint

[error] Replace ``Invalid·adjustment·percentage·for·schedule·${template[ll].name}`` with `⏎··········`Invalid·adjustment·percentage·for·schedule·${template[ll].name}`,⏎········`

(prettier/prettier)